sql/moz-fx-data-shared-prod/monitoring_derived/shredder_rows_deleted_v1/query.py

```diff
@@ -0,0 +1,138 @@
+"""Monitor the number of rows deleted by shredder."""
+import datetime
+from argparse import ArgumentParser
+from pathlib import Path
+from multiprocessing.pool import ThreadPool
+
+import yaml
+from google.api_core.exceptions import BadRequest
+from google.cloud import bigquery
+
+from bigquery_etl.util import standard_args
+
+JOB_INFO_QUERY = """
+WITH shredder AS (
+  SELECT DISTINCT
+    job_id AS full_job_id,
+    SPLIT(job_id, ":")[OFFSET(0)] AS project_id,
+    SPLIT(job_id, ".")[OFFSET(1)] AS job_id,
+  FROM
+    `moz-fx-data-shredder.shredder_state.shredder_state`
+),
+jobs AS (
+  -- https://cloud.google.com/bigquery/docs/information-schema-jobs
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shredder.region-us.INFORMATION_SCHEMA.JOBS_BY_PROJECT`
+  UNION ALL
+  SELECT
+    *
+  FROM
+    `moz-fx-data-bq-batch-prod.region-us.INFORMATION_SCHEMA.JOBS_BY_PROJECT`
```

Comment:
The `moz-fx-data-backfill-*` projects have also been used for Shredder jobs (as recently as last month), but I'm not sure unioning all those would work, as when I've tried to union a bunch of `INFORMATION_SCHEMA` queries together before I've sometimes run into a "too many metadata operations" kind of error.

There is an effort to make the `JOBS_BY_ORGANIZATION` view available ([DSRE-1106](https://mozilla-hub.atlassian.net/browse/DSRE-1106)) which would be useful for this, but I'm not sure what the ETA on that is.

Reply:
good point, I think I'll make it a cli arg and run the whole query once per project

Reply:
actually, after thinking: I'll get make a query to get the list of projects, then run a query per project, ensuring i get all the projects and don't run into `too many metadata operations`